src/components/input/_index.scss

```diff
@@ -9,6 +9,18 @@
 		@include scss.component-with-states("input-input", [focus]);
 	}
 
+	&.c-input__radio {
+		@include scss.component-properties("input-radio");
+
+		input[type="radio"] {
```

Comment:
@vgalatro It appears as a lint error. although the style is being applied, any chance you know what should be the definition for this, in order linting error disappear?

Reply:
@LauraPinilla , you don't need `[type="radio"]` here. Any `input` elements under `.c-input__radio` would be radio buttons anyway so that attribute selector is redundant.